### PR TITLE
Design changes2

### PR DESCRIPTION
**Pull request by andyiwest**
Update save/cancel buttons

### CODE REVIEW

File: _lib/cms/_tpl/default_edit.php [hunk 1]
```diff
@@ -115,8 +115,10 @@
 <table width="100%">
 <tr>
 	<td>
-		<button type="submit" style="font-weight:bold;">Save</button>
-		<button type="button" onclick="window.location.href='<?=$cancel_url;?>';">Cancel</button>
+    <div class="top-row">
```
Comment (adamjimenez) [inline _lib/cms/_tpl/default_edit.php]: indenting is off

File: _lib/cms/_tpl/default_edit.php [hunk 2]
```diff
@@ -225,8 +227,8 @@
 <table width="100%">
 <tr>
 	<td>
-		<button type="submit" style="font-weight:bold;">Save</button>
-		<button type="button" onclick="window.location.href='<?=$cancel_url;?>';">Cancel</button>
+    <button type="submit" class="btn btn-success">Save</button>
```
Comment (adamjimenez) [inline _lib/cms/_tpl/default_edit.php]: indenting is off here as well

**Comment by andyiwest** [comment]
That was strange, my Atom indents were fine. Copy/paste fail

**Comment by adamjimenez** [comment]
tabs vs spaces
File -> settings -> show invisibles

**Comment by andyiwest** [comment]
Cheers

**Pull request merged**